beacon-chain/db/kv/testing_helpers.go

```diff
@@ -0,0 +1,35 @@
+package kv
```

Comment:
```suggestion
package kv_testing
```

Can you avoid putting these testing methods in the production code path? For local testing, I think changing it to package_testing would work.

Reply:
Oh, you are using it in another package so this wont work.